### PR TITLE
Add technical governance mechanics description

### PR DESCRIPTION
**Pull request by pedroerp**
As discussed at VeloxCon 2024, this PR adds details about the technical governance mechanics, how it is structured, how it operates, and the decision making process behind the project.

The list of library components along with initial nominations will be shared soon.

### CODE REVIEW

**Review by xiaoxmeng — APPROVED** [review]
@pedroerp looks great. Thanks!

**Comment by pedroerp** [comment]
FYI: @majetideepak @assignUser @FelixYBW please help sharing with your groups

File: website/docs/community/technical-governance.md [hunk 1]
```diff
@@ -0,0 +1,278 @@
...
+
+This document describes the Velox technical governance structure, how it
+operates, and the decision-making process behind the project.
+
+## Summary
+
+The Velox community is composed of a set of highly-skilled individuals with the
+shared purpose of achieving the project mission. The project adopts a
+hierarchical technical governance structure, solely based on individuals and
+their merit in the community. Velox’s governing structure is composed of:
```
[the remark this answers is not in the document]
Comment (pedroerp) [reply website/docs/community/technical-governance.md:14]: I think "merit" is correct here, no?

File: website/docs/community/technical-governance.md [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+* Dispute local decisions when in conflict with project values, principles, or
+  direction.
+* Negotiating and resolving contentious issues, upkeeping the project’s best
+  interest.
+* Receiving broad requests for changes from project stakeholders and evaluating
+  them.
+  * Localized component-level requests are handled by component maintainers.
+* Nominate and confirm component maintainers and other PLC members.
+
+The project leadership council as a group has the power to dispute decisions
```
[the remark this answers is not in the document]
Comment (pedroerp) [reply website/docs/community/technical-governance.md:99]: I think "has" is also correct here. "As a group, it has the power to (...)"

**Review by xiaoxmeng — APPROVED** [review]

**Pull request closed without merging**